### PR TITLE
Update changelog for release 122

### PR DESCRIPTION
**Pull request by Gweaton**
(Tidies up the changelog, as I felt it was a bit too granular.)

- Update seeds and create data migration for adding ISPF fund entity to the service
- Add ISPF Level B UI user journeys for adding activities
- Add ISPF Level C/D UI user journeys for adding activities
- Inherit `is_oda` from the parent activity - Level C/D ISPF activities will therefore be ODA or non-ODA based on the (grand)parent Level B activity
- Do not allow removing the implementing organisation from a level C or D ISPF activity if it is the only one
- Require ISPF third-party projects (level D) to have at least one implementing organisation set through the new activity journey
- Hide ISPF-facing functionality in the service and exports when the feature flag is enabled
- Allow programmes to be redacted from IATI
- Set default values for currency and transaction type on Actual Transactions, output in IATI XML
- Remove reference to "ODA activities" in activity status descriptions

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [ ] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1120,28 +1120,23 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
 - Add Level B activity comments to budget exports
 - Strip leading/trailing whitespace and line breaks from comments in reports and exports
 
-## [unreleased]
+## [release-122]
```
Comment (yndajas) [inline CHANGELOG.md:1123]: Today's date?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1120,28 +1120,23 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
...
 
 - Update seeds and create data migration for adding ISPF fund entity to the service
-- Hide ISPF reports when the ISPF feature flag is enabled
-- ISPF programmes cannot be created when the feature flag is enabled
```
Comment (yndajas) [inline CHANGELOG.md:1127]: "ISPF programmes cannot be created when the feature flag is enabled" might not be covered explicitly now, but I think it's implied by "Hide ISPF-related entities in the service and exports when the feature flag is enabled"

**Review by yndajas — APPROVED** [review]
Looks good - just a couple of small things

File: CHANGELOG.md (hunk 2 above)
Comment (Gweaton) [reply CHANGELOG.md:1127]: Do you think "Hide ISPF-facing functionality in the service and exports when the feature flag is enabled" would be explicit enough?

**Pull request merged**